support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/input/ApproverAction.java

```diff
@@ -0,0 +1,25 @@
+package org.jenkinsci.plugins.workflow.support.steps.input;
+
+import hudson.model.InvisibleAction;
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+/**
+ * Created by valentinaarmenise on 21/01/2015.
+ */
```

Comment:
Can kill the dummy comment (use `@author` and/or add real Javadoc if you like), but please add a copyright header as in other source files.